src/components/layout.js

```diff
@@ -1,50 +1,34 @@
 import React from "react"
-import { useStaticQuery, Link, graphql } from "gatsby"
-import Navigation from "./navigation"
 import styled from "styled-components"
+import breakpoints from "./GlobalStyles/breakpoints"
 
-export default ({ children }) => {
-  const data = useStaticQuery(
-    graphql`
-          query {
-            site {
-              siteMetadata {
-                title
-              }
-            }
-          }
-        `
-  )
+const Layout = ({ children }) => {
 
   return (
     <Contianer>
-      <Navigation />
       {children}
     </Contianer >
   )
 }
 
 const Contianer = styled.main`
-  width: 100%;
-  padding-right: 1.5rem;
-  padding-left: 1.5rem;
+  width: 128rem;
+  max-width: 100%;
   margin-right: auto;
   margin-left: auto;
+  padding: 4rem;
 
-  @media (min-width: 576px) {
-    width: 54rem;
-  }
 
-  @media (min-width: 992px) {
-    width: 72rem;
+  @media ${breakpoints.device.maxXS} {
+    padding-right: 1.5rem;
```

Comment:
This is another optional thing but I'm trying to give you as much exposure/resources as possible in these PRs so you know what the alternatives are...

Instead of defining padding and margin etc manually throughout the app you could use something like [styled system](https://styled-system.com/) to take advantage of consistent spacing that can written as props against your components instead of having to re-write these all the time.